src/lib/ngsiNotify/senderThread.cpp

```diff
@@ -66,8 +66,6 @@ void* startSenderThread(void* p)
       std::string  out;
       int          r;
 
-      std::map<std::string, std::string> headers;
```

Comment:
Yes... a bit scary. Not only in this place... my IDE is complianing about unused variables that the build process is not able to warn.

Definitively, something smells bad in our CMakeList.txt stuff...